central/scannerdefinitions/handler/updater_test.go

```diff
@@ -77,6 +82,61 @@ func TestMappingUpdate(t *testing.T) {
 	assert.Equal(t, len(v4FileMapping), n)
 }
 
+func TestCvssUpdate(t *testing.T) {
+	filePath := filepath.Join(t.TempDir(), "test.tar.gz")
+	u := newUpdater(file.New(filePath), &http.Client{Timeout: 30 * time.Second}, cvssURL, 1*time.Hour)
+
+	// Should fetch first time.
+	require.NoError(t, u.doUpdate())
+	assertOnFileExistence(t, filePath, true)
+
+	n, err := countFilesInTarGz(filePath)
+	if err != nil {
+		t.Fatalf("Failed to count files in zip: %v", err)
+	}
+	assert.Greater(t, n, 21, "Number of files should be greater than 21")
```

Comment:
Will this always be 21, i.e. will the test break eventually? Do we need to check the file count here?

Reply:
It should be greater than 21 because the json files are from 2002 - current year. 
I count the file numbers to make sure this is updating the file that we desire

Reply:
I actually missed the fact that you're doing "Great" and not "Equal". Fine then.